drivers/hmis/app/models/hmis/hud/processors/custom_assessment_processor.rb

```diff
@@ -0,0 +1,20 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Hmis::Hud::Processors
+  class CustomAssessmentProcessor < Base
+    def factory_name
+      :owner_factory
+    end
+
+    def schema
+      Types::HmisSchema::CustomCaseNote
```

Comment:
This doesn't seem right>?

Reply:
Ah oops, thanks! Also added a fix for the test failure